### PR TITLE
Webapps deliverables

### PR DESCRIPTION
**Pull request by ylafon**
This add the more complete list of deliverable (including reference drafts) like in the Web Platform charter.

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
This seems fine, but we’d heard complaints about charters being too long (why we used the table instead). However, if this is the right way to go, and we don’t need to maintain the publication dates, I don’t mind tho.

**Review by dontcallmedom — APPROVED** [review]
I haven't reviewed the detailed data, but the overall change matches the issue I had brought up

**Comment by marcoscaceres** [comment]
@dontcallmedom, sorry, could you clarify or link to which issue you had raised? 

@ylafon looks good to merge if you are happy with it. I’ll run html tidy on it after.

**Review by siusin — APPROVED** [review]
LGTM. Thanks @ylafon !

File: webapps-2019.html [hunk 1]
```diff
@@ -884,7 +887,188 @@ <h2>
...
-      </section>
+	  </section>
+	  <section id="deliverablesfull">
+	  <h2>Detailed list of Deliverables</h2>
+	  <dl>
+	    <dt id=751 class='spec'><a href='https://www.w3.org/TR/FileAPI/' rel='versionof'>File API</a>
+	      <br>Latest publication: <a
+	      href='https://www.w3.org/TR/2018/WD-FileAPI-20181106/'>06 November
+	      2018</a></dt>
+	    <dd>Reference Draft:
```
Comment (plehegar) [inline webapps-2019.html:898]: Aren't you missing the CfE at
  https://lists.w3.org/Archives/Member/member-cfe/2013Sep/0004.html
?

Comment (ylafon) [reply webapps-2019.html:898]: I will seek confirmation, it is either a bug in the database or another rule that made that WD the reference draft, and not the FPWD.

Comment (ylafon) [reply webapps-2019.html:898]: So it should be an implicit cfe following the adoption of the specification.

Comment (plehegar) [reply webapps-2019.html:898]: do you mean: the document was adopted by a group after the CfE, thus making an implicit commitment?

**Pull request merged**